core/templates/pages/learner-group-pages/create-group/invite-learners.component.ts

```diff
@@ -89,8 +92,12 @@ export class InviteLearnersComponent {
     this.updateInvitedLearners();
   }
 
-  getProfileImageDataUrl(dataUrl: string): string {
-    return decodeURIComponent(dataUrl);
+  getProfileImagePngDataUrl(username: string): string {
+    return this.userService.getProfileImageDataUrlAsync(username);
+  }
+
+  getProfileImageWebpDataUrl(username: string): string {
+    return this.userService.getProfileImageDataUrlAsync(username, true);
```

Comment:
Yes, the reason being is that in case we have list of users for which we want to display the profile picture we are using this method. In this case in our HTML we are iterating over the users and then passing the username to the function so that we can fetch the profile picture of that specefic user.

Reply:
Hmm, right, makes sense.